bridge/engine/ActiveMediaList.cpp

```diff
@@ -355,12 +384,35 @@ void ActiveMediaList::process(const uint64_t timestampMs, bool& outDominantSpeak
 
     auto dominantSpeaker = heap.top();
 
+    if (!_c9_conference)
+    {
+        _activeTalkers.clear();
+    }
+
     for (size_t i = 0; i < _audioLastN && !heap.empty(); ++i)
     {
-        updateActiveAudioList(heap.top().participant);
+        auto top = heap.top();
```

Comment:
use auto& to avoid having a copy here. Not sure optimizer will fix that. I have seen "mistakes" I have made where it is by value if not explicitly specified.

Reply:
fixed! good catch